Assets/CodeBase/Entities/player/Player.cs

```diff
@@ -172,17 +176,13 @@ void FixedUpdate()
         if (playerRigidBody.velocity.y > 0.1)
         {
             _isGrounded = false;
+            _isFalling = false;
         }
-
         // player is falling
-        if (playerRigidBody.velocity.y < -0.1)
+        else if (playerRigidBody.velocity.y < -0.1)
```

Comment:
Out of curiosity, what is your rationale for making the limit -0.1 instead of 0?

Reply:
This is to check if the player is falling (which happens when the velocity is negative). It's 0 when the player is just standing on the ground. I would make another else-if for a ground check but isGrounded is already checked when checking for the collision with the floor, so there's no need for it

Reply:
Your if is a < rather than a <= though. Though in retrospect, the -0.1 can act as the small delay before Aristotle realizes he's falling.

Reply:
eh, let's go with = on the jump condition and leave it like this for falling